### DISCUSSION TITLE
Port Micropython to STM32H723ZGT6

### DISCUSSION THREAD

**Discussion by PYTRONIX-INC** [micropython/micropython#10245] [STM32 / Pyboard]
Hello,

I am trying to port micropython to STM32H723ZGT6. I am using the existing Nucleo H743Zi as a starting point https://github.com/micropython/micropython/tree/master/ports/stm32/boards/NUCLEO_H743ZI

can you please point to methodology I need to follow for doing the ports as I was not able to find any documentation on porting STM32 MCUs that are not part of the micopython repo. your help is much appreciate it!

**Comment by 4riceotter** [comment]
I found this page about a port to the NUCLEO-STB723ZG board, which may be what you need (at least to start with):

    https://micropython.org/download/NUCLEO_H723ZG/

I just got the board, but haven't got a chance to test it yet.  I'll be interested to find out how it goes for you.  Good luck!

Benjamin
